Dockerfile

```diff
@@ -8,10 +10,18 @@ RUN --mount=type=bind,from=ksmanis/gentoo-distcc:tcp,source=/var/cache/binpkgs,t
     emerge --info; \
     emerge distcc; \
     distcc --version; \
+    emerge ccache; \
+    ccache --version; \
+    echo "CCACHE_DIR=${CCACHE_DIR}" > /etc/env.d/02distcc-ssh-ccache; \
```

Comment:
Is this required? `CCACHE_DIR` will be automatically present in the container at runtime, as defined above.

Reply:
I tried without and in the case of ssh the value was lost. Maybe because `env-update` is used there for the `PATH` prepend. Could potentially move it to ssh-only so it gets ditched in case the ssh variant ever gets removed.

Reply:
That's odd; I would expect that a simple `export PATH=...` would suffice for `sshd` too. Anyway, thanks again for you effort, I'll have to do extensive testing to see what works and what not in order to give you accurate feedback. This will take some time!

Reply:
I will give it another try. That might well be something i never tried because i expected server env to not make it into client env, especially for things like PATH. But i never really tried and my expectation might be wrong. Plus there is a thing called `SetEnv` in `sshd_config(5)` to get rid of the `env-update` trick. But feel free to already test things.